test/performance/test_dmap.py

```diff
@@ -0,0 +1,44 @@
+import pydarn
+import cProfile, pstats, io
+import logging
+import os
+
+"""
+This is a simple testing script for the Dmap read and write
+methods. Warning: need a file to read in.
+"""
+logging.getLogger('pydarn').setLevel(logging.INFO)
+#@profile
```

Comment:
Instead of commenting this code out for mem profile, perhaps switch to a if __debug__: case or make some sort of conditionally dependent function.

Reply:
Applies for further cases too.

Reply:
Personally, I prefer this way of logging. Just makes the code cleaner and leaves the logging library to figure things out. 

I personally also keep logging INFO on because this is something a user would be using as well.

Reply:
Personally, I prefer this way of logging. Just makes the code cleaner and leaves the logging library to figure things out.

I personally also keep logging INFO on because this is something a user would be using as well.

Reply:
Maybe Im just misinterpretting this, but I was commenting on the commented out `#@profile
#def mem_profile():` part. I was thinking this could be within a conditional instead of commented out. The logging is fine.